File: src/main/java/no/stunor/origo/eventorapi/model/origo/entry/TeamMemberEntry.java
package no.stunor.origo.eventorapi.model.origo.entry;

import no.stunor.origo.eventorapi.model.event.CCard;
import no.stunor.origo.eventorapi.model.origo.CompetitorPerson;

public record TeamMemberEntry(
        CompetitorPerson person,
        int leg,
        CCard cCard) {
}

File: src/main/java/no/stunor/origo/eventorapi/model/event/entrylist/TeamMemberEntry.kt
package no.stunor.origo.eventorapi.model.event.entrylist

import no.stunor.origo.eventorapi.model.event.PunchingUnit
import no.stunor.origo.eventorapi.model.person.Gender
import no.stunor.origo.eventorapi.model.person.PersonName
import java.io.Serializable

data class TeamMemberEntry(
        var personId: String? = null,
        var name: PersonName? = null,
        var birthYear: Int? = null,
        var nationality: String? = null,
        var gender: Gender? = null,
        var punchingUnit: PunchingUnit? = null,
        var leg: Int = 1,
        var entryFeeIds: List<String> = listOf()
) : Serializable
